content/en/tracing/llm_observability/trace_an_llm_application.md

```diff
@@ -12,79 +12,172 @@ LLM Observability is not available in the US1-FED site.
 
-A given trace contains spans representing each choice made by an agent or each step of a workflow. A *span* represents some unit of work that your application is performing. Spans have a start time, duration, name, tags, and attributes.
+## Instrument your LLM application
 
-Multiple spans combine to form a trace, and a *root span* is the first span in a trace.
+<div class="alert alert-info">This guide uses the LLM Observability SDK for Python. If your application is not written in Python, you can complete the steps below with API requests instead of SDK function calls.</a></div>
 
-A trace can contain several kinds of spans. The *span kind* categorizes the type of work the span is performing. 
+Datadog provides [auto-instrumentation][4] to capture LLM calls for specific LLM provider libraries. However, manually instrumenting your LLM application using the Python SDK can unlock even more of Datadog's LLM Observability features.
 
-Only three span kinds can be the root span of a trace:
+To trace an LLM application:
 
-- **LLM span**: An individual LLM inference. LLM spans allow you to track inputs and outputs to your LLM calls; track tokens, error rates, and latencies for your LLM calls; and break down important metrics by models and model providers.
-- **Workflow span**: A grouping of LLM calls and their contextual operations, such as tool calls or preprocessing steps.
-- **Agent span**: A dynamic LLM workflow executed by an LLM agent.
+1. [Install the LLM Observability SDK][5].
+2. Configure the SDK by providing [the required environment variables][6] in your application startup command, or programmatically [in-code][7].
+    - Remember to configure your Datadog API key, Datadog site, and ML app name.
+3. [Create spans](#creating-spans) in your LLM application code to represent your application's operations.
+    - For additional examples and detailed usage, see the [Quickstart][8] and the [SDK documentation][9].
+    - [Nest spans](#nesting-spans) to create more useful traces.
+4. [Annotate your spans](#annotating-spans) with input data, output data, metadata (such as `temperature`), metrics (such as `prompt_tokens`), and key-value tags (such as `version:1.0.0`).
+5. Run your LLM application. 
+    - Use the `ddtrace-run` command if you used the command-line setup method, otherwise run your application as normal if you used the in-code setup method.
+6. Explore the resulting traces on the [LLM Observability traces page][10], and the resulting metrics on the out-of-the-box [LLM Observability dashboard][11].
 
-Different span kinds also have different parent-child relationships. For details, see [Span Kinds][4].
+### Creating spans
 
-## Instrument an LLM application
+To create a span, use the LLM Observability SDK's `ddtrace.llmobs.decorators.<SPAN_KIND>()` as a function decorator, replacing `<SPAN_KIND>` with the desired [span kind][4].
```

Comment:
Maybe this would be clearer if you edited it to be something like "You have two options for creating a span:

1. **Decorators**: Blah
2. **Inline**:  Blah